### PR TITLE
Toms branch

### PR DESCRIPTION
**Pull request by TomGroombridge**

### CODE REVIEW

File: spec/garage_spec.rb [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+  let(:bike) {Bike.new}
+  let(:van) {Van.new}
 
-	it "should fix bikes instantaneously once received" do
-	garage.dock(bike)
-	garage.fix_bikes		
-	end 
-end 
\ No newline at end of file
+  it "should accept a bike and fix it" do
```
Comment (TomGroombridge) [inline spec/garage_spec.rb:9]: this test is good but you have not checked the bike was broken to start with and is fixed now.

File: spec/garage_spec.rb [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
+    #expect garage to be empty
+    expect(garage.bike_count).to eq (0)
+    #lets dock a bike in
+    garage.accept(bike)
+    #we now expect garage to have 1 bike
+    expect(garage.bike_count).to eq (1)
+  end
+
+  it "should release a bike to the van after its fixed" do
```
Comment (TomGroombridge) [inline spec/garage_spec.rb]: you don't seem to actually fix the bike at all in here.

Comment (demecode) [reply spec/garage_spec.rb]: The accept method in bike.rb automatically fix the bike once docked in the garage

File: spec/garage_spec.rb (hunk 1 above)
Comment (demecode) [reply spec/garage_spec.rb:9]: I've now broken any bike that comes to the garage and once it has been docked in using 'accept' it should be fixed instantaneously

File: spec/garage_spec.rb (hunk 2 above)
Comment (TomGroombridge) [reply spec/garage_spec.rb]: 👍

File: lib/garage.rb [hunk 3]
```diff
@@ -1,11 +1,5 @@
-require_relative 'bike_container'
+require_relative "../lib/bike_container"
```
Comment (TomGroombridge) [inline lib/garage.rb:1]: if you include the module in the class you don't need to require the file path.

File: lib/van.rb [hunk 4]
```diff
@@ -1,7 +1,5 @@
-require_relative 'bike_container'
+require_relative "../lib/bike_container"
```
Comment (TomGroombridge) [inline lib/van.rb:1]: if you include the module in the class you don't need to require the file path.

Comment (demecode) [reply lib/van.rb:1]: Yes - makes sense - updated

File: lib/garage.rb (hunk 3 above)
Comment (demecode) [reply lib/garage.rb:1]: Yes - makes sense - updated

**Pull request merged**